config/settings/qa.yml

```diff
@@ -14,10 +14,5 @@ environment:
   selector_name: "qa"
 authentication:
   mode: persona   # none critical systems, ie localhost
-features:
-  rollover:
-    can_edit_current_and_next_cycles: true
```

Comment:
This shouldn't be on right now so removing too. It's set to false in settings

Reply:
oh yeah that was @frankieroberto doing